src/eureka/S1_detector_processing/bias_sub.py

```diff
@@ -0,0 +1,187 @@
+# Module for subtracting a super-bias image from science data sets
+# This is based on the superbias_step from the JWST pipeline v1.8.0
+# adapted by Kevin Stevenson, Feb 2023
+
+import numpy as np
+import logging
+from jwst.lib import reffile_utils
+from .group_level import mask_trace
+from ..lib.astropytable import savetable_S1
+from ..lib.smooth import smooth
+
+log = logging.getLogger(__name__)
+log.setLevel(logging.DEBUG)
+
+
+def do_correction(input_model, bias_model, meta, log):
+    """
+    Short Summary
+    -------------
```

Comment:
No need to have these lines

Reply:
I agree, but it was there when I copied this file from the `jwst` pipeline.  When I made edits to enable our custom bias correction, I left as much original code as possible.

Reply:
I still think we should remove this to keep uniformity between all of our other files.